### PR TITLE
#162948946 setup react boilerplate

### PR DESCRIPTION
**Pull request by Hesbon5600**
#### What does this PR do?
* It sets up an empty React project using Airbnb linting option.

#### Description of Task to be completed?
* Simply use `npm create-react-app .` on the existing repository.
* Configer `eslint`
#### How should this be manually tested?
* Clone the github repository `git clone https://github.com/andela/ah-fulldeck-frontend` on your local machine.
* Change your working directory to the project folder `cd ah-fulldeck-frontend`
* Install the required dependencies using `npm install`
* Run the local development server using `npm start` and you should get the following message:
  ```
  Compiled successfully!
  
  You can now view ah-fulldeck-frontend in the browser.
  
    Local:            http://localhost:3000/
    On Your Network:  http://10.10.160.30:3000/
  
  Note that the development build is not optimized.
  To create a production build, use npm run build
  ```

#### Any background context you want to provide?
Create React App now supports the use of different environment variables files out of the box. For
more information click on this link: [here](https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d).

Therefore, other than creating the relevant files, nothing more needs to be done to configure their support.

#### What are the relevant pivotal tracker stories?
[#162948946](https://www.pivotaltracker.com/story/show/162948946)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (victorjambo) [inline public/index.html]: Good job on this PR. Can you change this from `React App` the name of the app. @Hesbon5600

**Review by charitymarani — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (Hesbon5600) [reply public/index.html]: Resolved

File: src/logo.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
...1 3.2-26.9 5.9-41.2 8 4.9-7.7 9.8-15.6 14.4-23.7 4.6-8 8.9-16.1 13-24.1zM421.2 430c-9.3-9.6-18.6-20.3-27.8-32 9 .4 18.2.7 27.5.7 9.4 0 18.7-.2 27.8-.7-9 11.7-18.3 22.4-27.5 32zm-74.4-58.9c-14.2-2.1-27.9-4.7-41-7.9 3.7-12.9 8.3-26.2 13.5-39.5 4.1 8 8.4 16 13.1 24 4.7 8 9.5 15.8 14.4 23.4zM420.7 163c9.3 9.6 18.6 20.3 27.8 32-9-.4-18.2-.7-27.5-.7-9.4 0-18.7.2-27.8.7 9-11.7 18.3-22.4 27.5-32zm-74 58.9c-4.9 7.7-9.8 15.6-14.4 23.7-4.6 8-8.9 16-13 24-5.4-13.4-10-26.8-13.8-39.8 13.1-3.1 26.9-5.8 41.2-7.9zm-90.5 125.2c-35.4-15.1-58.3-34.9-58.3-50.6 0-15.7 22.9-35.6 58.3-50.6 8.6-3.7 18-7 27.7-10.1 5.7 19.6 13.2 40 22.5 60.9-9.2 20.8-16.6 41.1-22.2 60.6-9.9-3.1-19.3-6.5-28-10.2zM310 490c-13.6-7.8-19.5-37.5-14.9-75.7 1.1-9.4 2.9-19.3 5.1-29.4 19.6 4.8 41 8.5 63.5 10.9 13.5 18.5 27.5 35.3 41.6 50-32.6 30.3-63.2 46.9-84 46.9-4.5-.1-8.3-1-11.3-2.7zm237.2-76.2c4.7 38.2-1.1 67.9-14.6 75.8-3 1.8-6.9 2.6-11.5 2.6-20.7 0-51.4-16.5-84-46.6 14-14.7 28-31.4 41.3-49.9 22.6-2.4 44-6.1 63.6-11 2.3 10.1 4.1 19.8 5.2 29.1zm38.5-66.7c-8.6 3.7-18 7-27.7 10.1-5.7-19.6-13.2-40-22.5-60.9 9.2-20.8 16.6-41.1 22.2-60.6 9.9 3.1 19.3 6.5 28.1 10.2 35.4 15.1 58.3 34.9 58.3 50.6-.1 15.7-23 35.6-58.4 50.6zM320.8 78.4z"/>
```
Comment (Nicanor008) [inline src/logo.svg:3]: Unnecessary logo, but I guess this should be resolved by another chore to our author's Haven logo. Setup works perfectly

**Review by victorjambo — APPROVED** [review]
LGTM

**Review by JoshuaKodhe — APPROVED** [review]

**Pull request merged**